assets/block/init.php

```diff
@@ -87,6 +87,6 @@ function bctt_block_callback( $attributes ) {
 		'nofollow' => $nofollow ? 'yes' : 'no',
 		'prompt'   => $prompt
 	), $attributes );
-
+    bctt_scripts();
```

Comment:
This will break shortcodes!, the ideal place to move this would be to shortcode here 

https://github.com/Benunc/better-click-to-tweet/blob/0f4a7d76c3378c87967c87eac1efb76052c31e32/better-click-to-tweet.php#L94